server/db/models/user.js

```diff
@@ -34,6 +34,10 @@ export default (sequelize, DataTypes) => {
       profilePic: {
         type: DataTypes.STRING,
         allowNull: true
+      },
+      role: {
+        type: DataTypes.ENUM('admin', 'user'),
```

Comment:
I can see you define two roles while you use three. Where was "super_admin" defined?